src/skip-to-content/skip-to-content.scss

```diff
@@ -15,23 +15,26 @@
   position: fixed;
   text-align: center;
   left: 50%;
+  opacity: 0;
   top: $iui-baseline * 2;
   transform: translateX(-50%) translateY(-170%);
   transition: background-color $iui-speed-fast ease-in-out;
   @media (prefers-reduced-motion: no-preference) {
-    transition: background-color $iui-speed-fast ease-in-out, transform $iui-speed-slow ease-in-out,
-      box-shadow $iui-speed-fast ease-in-out;
+    transition: opacity $iui-speed-slow ease-in-out, background-color $iui-speed-fast ease-in-out,
+      transform $iui-speed-slow ease-in-out, box-shadow $iui-speed-fast ease-in-out;
   }
 
   &:hover {
     @include iui-blur($opacity: 2);
   }
 
   &:focus {
+    opacity: 1;
     transform: translateX(-50%) translateY(0);
     box-shadow: $iui-elevation-16;
     @media (prefers-reduced-motion: no-preference) {
-      transition: transform $iui-speed-fast ease-in-out;
+      transition: opacity $iui-speed-fast ease-in-out, background-color $iui-speed-fast ease-in-out,
```

Comment:
I think these should have a delay of `0` so that they're instantly visible

Reply:
Here's a screen recording with reduced motion on & then reduced motion off.  It is instant when reduced motion is enabled.
![Screen Recording 2022-03-28 at 4 14 35 PM](https://user-images.githubusercontent.com/849817/160479852-b27d6dec-a1c0-4c96-944f-cca8aa2d784b.gif)

Reply:
I'm talking about the opacity and background-color even when reduced-motion is off. I think @elephantcatdog intended for the link to be fully visible when it starts sliding into view